book/src/import.md

```diff
@@ -0,0 +1,87 @@
+# Import
+
+The `import` directive allows you to include type definitions and other declarations from other `.zng` files into your main specification. Types, traits, and modules can appear multiple times across the transitive set of imported files, and their content is merged together.
+
+## Syntax
+
+```zng
+import "path/to/file.zng";
+```
+
+## Path Resolution
+
+Import paths are resolved relative to the directory containing the current `.zng` file:
+
+- `import "./types.zng";` or
+- `import "subdir/types.zng";`
```

Comment:
What do you think about forcing the "./foo/bar.zng" and reserving "foo/bar.zng" similar to js/npm? We can later use `foo/bar.zng` for `bar.zng` file of the `foo` crate.

Reply:
Good call. Done!